crates/icrate/src/GameController/fixes/mod.rs

```diff
@@ -0,0 +1,86 @@
+use crate::common::c_long;
+use crate::Foundation::NSString;
+
+pub type GCKeyCode = c_long;
```

Comment:
Add note about `CFIndex` being `c_long_long` on `__LLP64__` / Windows 64-bit (and how this doesn't matter for us)